source/css/_common/outline/sidebar/sidebar-dimmer.styl

```diff
@@ -2,7 +2,7 @@
   display: none;
 }
 
-+mobile() {
++tablet-mobile() {
```

Comment:
Perhaps unifying the behavior of the tablet and desktop is a better choice?

Reply:
I don't think "Tablet" has enough space for both the article and sidebar.

Reply:
This problem is a bit tricky, because both 7.9-inch iPad mini and 12.9-inch iPad Pro are "Tablet"s.

I will merge this pull request and make adjustments based on subsequent user feedback. Thanks for your contribution.